packages/pangea-sdk/PangeaCyber.Net/AIGuard/Models/MaliciousEntity.cs

```diff
@@ -0,0 +1,24 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace PangeaCyber.Net.AIGuard.Models;
+
+/// <summary>Represents a malicious entity found in the text.</summary>
+[JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
+public sealed class MaliciousEntity
+{
+    /// <summary>The type of malicious entity.</summary>
+    public string Type { get; set; } = default!;
+
+    /// <summary>The value of the malicious entity.</summary>
+    public string Value { get; set; } = default!;
+
+    /// <summary>Whether the entity was redacted.</summary>
+    public bool? Redacted { get; set; }
+
+    /// <summary>The starting position of the entity in the text.</summary>
+    public float? StartPos { get; set; }
```

Comment:
Should it be an int?

Reply:
OpenAPI spec uses the wider `number` type but sure, updated.